### ISSUE TITLE
[Accessibility]: Search contexts management screen reader issues

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#34740]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34191

### Problem description

The following issues exists in the search context management page (https://sourcegraph.com/contexts):

- [ ] In the list page, "Your search contexts" read as a button when it should be a tab and read as selected
![image](https://user-images.githubusercontent.com/206864/166081098-f6777b8c-403a-4b38-91b5-2de9d5041d85.png)
- [ ] On the add/edit page, middle dot under description should not be read by screen reader
![CleanShot 2022-04-29 at 16 41 24](https://user-images.githubusercontent.com/206864/166081130-d3fb5175-f266-451d-ba54-285790b3eb57.png)
- [ ] On the add/edit page, horizontal splitters should not be ready by screen reader
![CleanShot 2022-04-29 at 16 41 57](https://user-images.githubusercontent.com/206864/166081156-13263b46-78e1-4dce-8815-522e1925a7ab.png)
- [ ] On the add/edit page, public/private radio buttons labels are too long, should only be "Public" or "Private"
![CleanShot 2022-04-29 at 16 41 57](https://user-images.githubusercontent.com/206864/166081167-8e63c990-5d71-4839-adf3-288663cc80fd.png)

### Expected behavior

Should be fixed as described above

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by gitstart-sourcegraph** [comment]
Hi @limitedmage 
For this requirement: ` On the add/edit page, public/private radio buttons labels are too long, should only be "Public" or "Private"`,
Are the arrow locations correct in the image below?, 
Would it be correct to assume that you meant to point out the circled sections in the image.
 If yes, could you also clarify assumption that the requirement is to hide the circled helper text from screen reader?
![168582351-d16d56ee-dd01-4ae2-a999-344d3aecf5f4](https://user-images.githubusercontent.com/89894075/168749613-76eb2605-c8a1-4f50-bc52-5f2abfd204a0.png)

**Comment by gitstart-sourcegraph** [comment]
Yea that makes sense @limitedmage, Thanks
